string rtov_key
---
race_perception_msgs/SITOV[] sitovs

